python/src/aiconfig/editor/client/src/components/global/ShareButton.tsx

```diff
@@ -1,12 +1,43 @@
-import { Button, Tooltip } from "@mantine/core";
+import { Button, Modal, Tooltip, createStyles, rem } from "@mantine/core";
 import { memo, useState } from "react";
+import { useDisclosure } from "@mantine/hooks";
+import CopyButton from "../CopyButton";
 
 type Props = {
   onShare: () => Promise<string | void>;
 };
 
+const useStyles = createStyles((theme) => ({
+  copyIcon: {
+    alignItems: "flex-end",
+    align: "right",
+    justifyContent: "flex-end",
+    position: "absolute",
+    right: 0,
+  },
+  urlContainer: {
+    display: "flex",
+    flexDirection: "row",
+    width: "100%",
+    align: "center",
+    [theme.fn.smallerThan("sm")]: {
+      marginLeft: "0",
+      display: "block",
+      position: "static",
+      bottom: -10,
+      left: 0,
+      height: 28,
+      margin: "10px 0",
+    },
```

Comment:
What does this look like on small screens?

Reply:
It doesn't actually apply to us since we can never go smaller than certain width for some reason:


https://github.com/lastmile-ai/aiconfig/assets/151060367/3401fb87-d224-498b-a296-b315f8ba1e4e